src/util/logger/Logger.cpp

```diff
@@ -47,8 +47,10 @@ void Logger::log(std::string message, const std::string log_type) {
     long millis = tv.tv_sec * 1000 + tv.tv_usec / 1000;
     // TODO: This temporarily fixes spdlog hanging after forking. This will prevent using the actual logger and simulate
     // the behavior using cout instead. But it will not write logs to the log file.
-    cout << " [" << millis << "] [" << log_type << "] [" << worker_name << " : " << getpid() << ":" << tid << "] "
-         << message << endl;
+    if (log_type.compare("debug") != 0) {
+        cout << " [" << millis << "] [" << log_type << "] [" << worker_name << " : " << getpid() << ":" << tid << "] "
```

Comment:
Shall we move this `cout` statement also to a debug log line?

Reply:
If we do, it will reduce the performance. We need to avoid debug logs in the default setup.

Reply:
Ok